src/bci_build/package/postfix.py

```diff
@@ -0,0 +1,92 @@
+from bci_build.package import Replacement
+from bci_build.package import SupportLevel
+
+from .appcollection import ApplicationCollectionContainer
+
+_POSTFIX_FILES = {}
+for filename in (
+    "entrypoint.sh",
+    "entrypoint.sles.sh",
+    "smtpd_sender_login_maps",
+    "virtual_alias_domains",
+    "virtual_alias_maps",
+    "virtual_gid_maps",
+    "virtual_mailbox_maps",
+    "virtual_uid_maps",
+):
+    if filename.startswith("entrypoint."):
+        _POSTFIX_FILES[filename] = (
+            Path(__file__).parent / "postfix" / "entrypoint" / filename
+        ).read_bytes()
+    else:
+        _POSTFIX_FILES[filename] = (
+            Path(__file__).parent / "postfix" / "entrypoint" / "ldap" / filename
+        ).read_bytes()
+
+
+POSTFIX_CONTAINERS = [
+    ApplicationCollectionContainer(
+        name="postfix",
+        package_name="postfix-image"
+        if os_version.is_tumbleweed
+        else "sac-postfix-image",
+        pretty_name="Postfix",
+        custom_description="Postfix container is fast and secure mail server, {based_on_container}.",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION and os_version.is_tumbleweed,
+        version="%%postfix_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%postfix_version%%",
+                package_name="postfix",
+                parse_version="minor",
+            )
+        ],
+        package_list=[
+            name
+            for name in (
+                "ca-certificates-mozilla",
+                "cyrus-sasl",
+                "cyrus-sasl-plain",
+                "ed",
+                "netcfg",
+                "postfix",
+                "postfix-ldap",
+                "postfix-lmdb",
+                "timezone",
+            )
+            + (
+                (
+                    "mandoc",
+                    "spamassassin-spamc",
+                    "spamass-milter",
+                )
+                if os_version == OsVersion.TUMBLEWEED
+                else ()
+            )
+        ],
+        entrypoint=["/entrypoint/entrypoint.sh"],
+        cmd=["postfix", "start"],
+        extra_files=_POSTFIX_FILES,
+        support_level=SupportLevel.TECHPREVIEW,
+        exposes_tcp=[25, 465, 587],
+        volumes=["/var/spool/postfix", "/var/spool/vmail", "/etc/pki"],
+        custom_end=f"""{DOCKERFILE_RUN} mkdir -p /entrypoint/ldap
+COPY {"entrypoint.sh" if os_version == OsVersion.TUMBLEWEED else "entrypoint.sles.sh"} /entrypoint/entrypoint.sh
+{DOCKERFILE_RUN} chmod +x /entrypoint/entrypoint.sh
+COPY smtpd_sender_login_maps virtual_alias_domains virtual_alias_maps virtual_gid_maps virtual_mailbox_maps virtual_uid_maps /entrypoint/ldap/
+HEALTHCHECK --interval=5s --timeout=10s --start-period=30s --retries=3 \
+        CMD postfix status
+""",
+    )
+    for os_version in set(ALL_NONBASE_OS_VERSIONS) | {OsVersion.SLE16_0}
```

Comment:
@dirkmueller, LGTM

Just the following bit.
Currently there's no image package getting created for 15.5 in https://build.opensuse.org/project/show/devel:BCI:SLE-15-SP5.

And one of the CI tests in https://github.com/SUSE/BCI-tests/pull/487 , is [complaining for the missing image.](https://github.com/SUSE/BCI-tests/actions/runs/10147250089/job/28058117661?pr=487#step:12:700)

```
Error: No such object: registry.opensuse.org/devel/bci/sle-15-sp5/containerfile/suse/postfix:3.7
Error response from daemon: name unknown
```

```suggestion
    for os_version in ALL_BASE_OS_VERSIONS
```

Reply:
With the release of SLES 15 SP6, SLE15 SP5 is really end of life in regards to new application containers. so the state of having it only for SP6 is correct. 

Also there won't be appcollection containers for SP5.

Reply:
> With the release of SLES 15 SP6, SLE15 SP5 is really end of life in regards to new application containers. so the state of having it only for SP6 is correct.

Ack. Approved the changes.